moment/moment-node.d.ts

```diff
@@ -292,6 +304,9 @@ declare module moment {
         isSame(b: Moment | string | number | Date | number[], granularity?: string): boolean;
         isBetween(a: Moment | string | number | Date | number[], b: Moment | string | number | Date | number[], granularity?: string): boolean;
 
+        // Since version 2.10.7+
+        isSameOrBefore(b: Moment | string | number | Date | number[], granularity?: string): boolean;
```

Comment:
consider making a type alias (outside of this `Moment` interface) since this union is used 6 times now. something like

```ts
type MomentComparable = Moment | string | number | Date | number[];
```